tools/stress_mcp.py

```
@@ -0,0 +1,262 @@
+            stats["errors"] += 1
+            await asyncio.sleep(0.2)
+            continue
+        finally:
+            try:
+                writer.close()  # type: ignore
+                await writer.wait_closed()  # type: ignore
+            except Exception:
+                pass
+
+
+async def reload_churn_task(project_path: str, stop_time: float, unity_file: str | None, host: str, port: int, stats: dict):
```

Ruff (0.12.2)

Unused function argument: `project_path`

(ARG001)